go/common/enclave.go

```diff
@@ -33,9 +33,6 @@ type Enclave interface {
 	// InitEnclave - initialise an enclave with a seed received by another enclave
 	InitEnclave(secret EncryptedSharedEnclaveSecret) error
 
-	// ProduceGenesis - produce the genesis batch
-	ProduceGenesis(blkHash gethcommon.Hash) (*ExtBatch, error)
```

Comment:
We now produce the genesis as part of processing a block as the sequencer, rather than producing it ahead of time, submitting it to the L1, and reading it back as a rollup.